micrometer-observation/src/main/java/io/micrometer/observation/docs/DocumentedObservation.java

```diff
@@ -201,14 +201,14 @@ default <T extends Observation.Context> Observation observation(@Nullable Observ
      * pre-configured conventions
      * @param defaultConvention default convention that will be picked if there was
      * neither custom convention nor a pre-configured one via
-     * {@link ObservationRegistry.ObservationConfig#observationConvention(GlobalObservationConvention[])}
+     * {@link ObservationRegistry.ObservationConfig#observationConventions(GlobalObservationConvention[])}
      * @param contextSupplier observation context supplier
      * @param registry observation registry
      * @return observation
      */
-    default <T extends Observation.Context> Observation createNotStarted(
-            @Nullable ObservationConvention<T> customConvention, @NonNull ObservationConvention<T> defaultConvention,
-            @NonNull Supplier<T> contextSupplier, @NonNull ObservationRegistry registry) {
+    default <T extends Observation.Context> Observation observation(@Nullable ObservationConvention<T> customConvention,
```

Comment:
Let's discuss if we want to rename this class.
@shakuzen Did you get some feedback on this?

Reply:
I thought we rename existing `observation` to `createNotStarted`?
Either way, it is better to be unified.

Reply:
Yepp, that's why the let's discuss comment.
I think:
1. If we want to keep the class name as-is: it is better to rename everything to `createNotStarted`
2. If we want to rename the class to `ObservationDocumentation` (or similar): it is better to rename everything to `observation`
3. If we want to keep changes minimal: what I did

I think the best would be # 2 but it is also the most invasive.

Reply:
I think `ObservationDocumentation` better expresses what the interface is for and makes it clear that it is not an `Observation` itself. The feedback I got so far was that they agreed with that. I think it will be easier to consider that change in a separate pull request, though, so we don't block making the rest of these changes.